primitives/version/versions.go

```diff
@@ -20,28 +20,23 @@
 
 package version
 
-import (
-	"encoding/binary"
-)
-
 const (
-	Phase0 uint32 = iota
-	Altair
-	Bellatrix
-	Capella
-	Deneb
-	DenebPlus
-	Electra
+	// Phase0 is the first version of the Beacon Chain.
+	Phase0 uint32 = 0
+	// Altair is the first hardfork of the Beacon Chain.
+	Altair uint32 = 1
+	// Bellatrix is the second hardfork of the Beacon Chain.
+	Bellatrix uint32 = 2
+	// Capella is the third hardfork of the Beacon Chain.
+	Capella uint32 = 3
+	// Deneb is the first version of Deneb, used for genesis of Berachain mainnet.
+	Deneb uint32 = 4
+	// Deneb1 is the first hardfork of Deneb on Berachain mainnet (TBD if used).
+	// There may also be Deneb2, Deneb3, etc. hardforks.
+	Deneb1 uint32 = 260
```

Comment:
So technically this is a breaking change since we are reordering the forks. It should be fine since DenebPlus was never active.
@calbera can you confirm this?

Reply:
correct